File: db4on/reference/db4o-reference-chapters/src/com/db4odoc/f1/diagnostics/Empty.cs
/* Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com */
using System;

namespace com.db4odoc.f1.diagnostics
{
	public class Empty
	{
		public Empty()
		{
		}

		public string CurrentTime()
		{
			DateTime dt = DateTime.Now;
			String time = dt.ToString("d");
			return time;
		}

		override public string ToString()
		{
			return CurrentTime();
		}
	}
}

File: reference.vb.net/Db4objects.Db4odoc/Diagnostics/Empty.vb
' Copyright (C) 2004 - 2006 db4objects Inc. http://www.db4o.com 
Imports System

Namespace Db4objects.Db4odoc.Diagnostics
    Public Class Empty
        Public Sub New()
        End Sub

        Public Function CurrentTime() As String
            Dim dt As DateTime = DateTime.Now
            Dim time As String = dt.ToString("d")
            Return time
        End Function

        Public Overrides Function ToString() As String
            Return CurrentTime()
        End Function
    End Class
End Namespace
